examples/platform/silabs/efr32/rs911x/rsi_if.c

```diff
@@ -685,7 +702,8 @@ void ProcessEvent(WfxEvent_t inEvent)
         // is independant of IP connectivity.
         break;
     case WFX_EVT_STA_DISCONN:
-        // TODO: This event is not being posted anywhere, seems to be a dead code or we are missing something
+        // TODO: This event is not being posted anywhere, seems to be a dead code or we are missing
```

Comment:
Todo has been there for a while. Do we need it to capture this event here or not? if not delete it.

Reply:
removed the TODOs